knowledge-base/error-resource-not-found-after-finishing-the-item-template-wizard.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Error 404 Resource not found
+description: After finishing the item template wizard for creating HTML5-based report viewer with either separate hosing of the Reporting REST Service or Report Server connection, an error 404 is thrown in the browser console.
```

Comment:
I suggest something like: "After creating an HTML5-based report viewer with separate Reporting REST Service or Report Server with the VS item template wizard, a 404 error is thrown in the browser console."
Mind the type in "hosing"->"hosting"

Reply:
agree